### PR TITLE
Activation term exponent for I_NaP and I_T configurable in ht_neuron

### LINKED ISSUE #1189: Activation particles for NaP and T in ht_neuron
In setting up a network of ht_neuron neurons I've found it necessary to change the number of activation particles for the NaP and T currents. I've done this by altering the code for ht_neuron as shown in the attached file, where both currents now have just one activation particle. But more generally it would be convenient if the number of activation particles for these currents could be made settable.

[ht_neuron_sh_DKhep.cpp.txt](https://github.com/nest/nest-simulator/files/3151380/ht_neuron_sh_DKhep.cpp.txt)

### PR DESCRIPTION
**Pull request by heplesser**
This PR fixes #1189.

The PR makes the activation term exponents in the HH-formulation of I_NaP and I_T of the ht_neuron configurable, keeping the previously hard-coded values as defaults. I decided not to enforce integer values but allow doubles since it is mathematically well-defined and permits greater flexibility to the modeller. Parameter names `N_NaP` and `N_T` are chosen to match the "model details" notebook on the HT model.

I also updated the model details notebook correspondingly and added additional information on the NMDA channel implementation based on a recent discussion in the mailing list.

@ricardomurphy Could you review if this matches your expectations? Github does not allow me to add you explicitly as a reviewer, unfortunately.

### CODE REVIEW

**Comment by heplesser** [comment]
@ricardomurphy confirmed by email that the PR meets his expectations for a solution to #1189.

File: doc/model_details/HillTononiModels.ipynb [hunk 1]
```diff
@@ -307,8 +313,28 @@
...
+    "**Note**: NEST also supports instantaneous NMDA dynamics using a boolean switch. In that case $m(V, t)=m^{\\infty}(V)$. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### Detailed relation between NMDA channel model in NEST and previous work\n",
+    "\n",
+    "The NMDA channel dynamics are not very clearly described in the original paper by Hill and Tononi. The model implemented in NEST is in this point mostly based on inspecting the code of Sean Hill’s Synthesis simulator. \n",
```
Comment (stinebuu) [inline doc/model_details/HillTononiModels.ipynb:325]:
```suggestion
    "The NMDA channel dynamics are not very clearly described in the original paper by Hill and Tononi. The model implemented in NEST is at this point mostly based on inspecting the code of Sean Hill’s Synthesis simulator. \n",
```

**Review by stinebuu — COMMENTED** [review]
Looks good to me! I just have a tiny suggestion.

**Review by stinebuu — APPROVED** [review]
Thanks!

**Review by pnbabu — APPROVED** [review]
Looks good to me!

**Pull request merged**